integration-tests/discovery-tests/order-consumer/src/main/java/com/huaweicloud/sample/GovernanceController.java

```diff
@@ -232,4 +236,16 @@ public String restTemplateInstanceBulkhead() throws Exception {
   public String loadbalance() {
     return feignService.loadbalabce();
   }
+
+  @GetMapping("/testFeignFaultInjection")
+  public String testFeignFaultInjection(@RequestParam String name) {
+    webfluxService.sayHello(name);
+    return feignService.testFaultInjection();
+  }
+
+  @GetMapping("/testTemplateFaultInjection")
+  public String testTemplateFaultInjection(@RequestParam String name) {
+    restTemplate.getForObject("http://webflux/sayHello?name=tom", String.class);
```

Comment:
whats `restTemplate.getForObject("http://webflux/sayHello?name=tom", String.class);'` for ?